dom.bs

```diff
@@ -1419,6 +1419,103 @@ that gave folks all the wrong ideas. <a>Events</a> do not represent or cause act
 can only be used to influence an ongoing one.
 
 
+<h3 id=controlling-promises>Controlling {{Promise}}s</h3>
+
+Though {{Promise}} objects don't have any built-in cancellation mechanism, many {{Promise}}-vending
```

Comment:
I'm going to change everything to `cancel()`/cancellation. I used `abort()` because that's what folks in https://github.com/whatwg/fetch/issues/447 were saying, but I'm having trouble getting past "abortion"'s other common usage while typing it out.